WPMedia/fusion-news-theme-blocks
# TMEDIA-460: Confirm Password component

## Description
Add confirm password form component functionality

## Jira Ticket
- [TMEDIA-460](https://arcpublishing.atlassian.net/browse/TMEDIA-460)

## Acceptance Criteria
* FormComparePassword checks if the Password/Confirm Password fields contain identical inputs
* If the fields do not match, the error state is shown

## Test Steps
1. Checkout this branch `git checkout TMEDIA-460-confirm-password`
2. Run fusion repo with linked blocks `npm run storybook`
3. Validate the confirm password items work as intended:
![image](https://user-images.githubusercontent.com/2287238/130528822-bce3d845-0d5c-4f4e-9620-a39b3680c9a7.png)
![image](https://user-images.githubusercontent.com/2287238/130528834-93430550-d1a7-4d7b-a271-5b461c628dfc.png)
![image](https://user-images.githubusercontent.com/2287238/130528861-6bf131c4-dc58-41b1-a62c-c1c7a0d94125.png)

## Author Checklist
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
